sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -2044,10 +2043,10 @@ def test_same_radius_neighbors_parallel(algorithm):
     assert_allclose(graph, graph_parallel)
 
 
-@pytest.mark.parametrize("backend", JOBLIB_BACKENDS)
+@pytest.mark.parametrize("backend", ["threading", "loky"])
```

Comment:
we don't want to test multiprocessing and sequential backends?

Reply:
The multiprocessing case is very slow on macOS (and maybe windows as well): this backend creates new processes from scratch each time which is can take several tens of seconds to run on the overloaded macOS Azure CI.

I think the loky backend is enough to test for multiprocess-related problems in our Cython code. The main difference between multiprocessing and loky is that loky reuses its workers across calls making it much more efficient on overloaded machines.

Reply:
Also sequential is already tested everywhere else in `test_neighbors.py` (as it's the default backend for the default value of `n_jobs`).